end_to_end_tests/generated_code_live_tests/test_defaults.py

```diff
@@ -0,0 +1,98 @@
+
```

Comment:
The tests in this file make some of the current low-level unit tests in `test_any`, `test_boolean`, etc. unnecessary in terms of coverage.